<!--
	description
-->

<launch>

	<!-- <arg name="cmd_vel_topic" default="/cmd_vel" /> -->
	<arg name="odom_topic" default="odom_converted" />
	<!-- <arg name="odom_topic" default="base_footprint" /> -->
	<!-- <arg name="move_forward_only" default="false"/> -->

	<node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen" >
		<param name="base_local_planner" value="dwa_local_planner/DWAPlannerROS" />
		<rosparam file="$(find slam_navigation)/param_AMR_gazebo/costmap_common_params_burger.yaml" command="load" ns="global_costmap" />
		<rosparam file="$(find slam_navigation)/param_AMR_gazebo/costmap_common_params_burger.yaml" command="load" ns="local_costmap" />
		<rosparam file="$(find slam_navigation)/param_AMR_gazebo/local_costmap_params.yaml" command="load" />
		<rosparam file="$(find slam_navigation)/param_AMR_gazebo/global_costmap_params.yaml" command="load" />
		<rosparam file="$(find slam_navigation)/param_AMR_gazebo/move_base_params.yaml" command="load" />
		<rosparam file="$(find slam_navigation)/param_AMR_gazebo/dwa_local_planner_params_burger.yaml" command="load" />
			<!-- <remap from="cmd_vel" to="$(arg cmd_vel_topic)"/> -->
			<remap from="odom" to="$(arg odom_topic)"/>
			<!-- <param name="DWAPlannerROS/min_vel_x" value="0.0" if="$(arg move_forward_only)" /> -->

		<!-- <rosparam file="$(find slam_navigation)/param1/wsl.yaml" command="load" /> -->
  	</node>

</launch>
